document/paragraphproperties.go

```diff
@@ -199,3 +199,186 @@ func (p ParagraphProperties) SetHangingIndent(m measurement.Distance) {
+func (p ParagraphProperties) Caps() bool {
+	return getBool(p.x.RPr.Caps)
+}
+
+// Strike returns true if paragraph is striked.
+func (p ParagraphProperties) Strike() bool {
+	return getBool(p.x.RPr.Strike)
+}
+
+// DoubleStrike returns true if paragraph is double striked.
+func (p ParagraphProperties) DoubleStrike() bool {
+	return getBool(p.x.RPr.Dstrike)
+}
+
+// Outline returns true if paragraph outline is on.
+func (p ParagraphProperties) Outline() bool {
+	return getBool(p.x.RPr.Outline)
+}
+
+// Shadow returns true if paragraph shadow is on.
+func (p ParagraphProperties) Shadow() bool {
+	return getBool(p.x.RPr.Shadow)
+}
+
+// Emboss returns true if paragraph emboss is on.
+func (p ParagraphProperties) Emboss() bool {
+	return getBool(p.x.RPr.Emboss)
+}
+
+// RightToLeft returns true if paragraph text goes from right to left.
+func (p ParagraphProperties) RightToLeft() bool {
+	return getBool(p.x.RPr.Rtl)
+}
+
+// RStyle returns the name of paragraph runs style.
+func (p ParagraphProperties) RStyle() string {
+	if p.x.RPr.RStyle != nil {
+		return p.x.RPr.RStyle.ValAttr
+	}
+	return ""
+}
+
+// Font returns the name of paragraph font family.
+func (p ParagraphProperties) Font() string {
+	if fonts := p.x.RPr.RFonts; fonts != nil {
+		if fonts.AsciiAttr != nil {
+			return *fonts.AsciiAttr
+		} else if fonts.HAnsiAttr != nil {
+			return *fonts.HAnsiAttr
+		} else if fonts.CsAttr != nil {
+			return *fonts.CsAttr
+		}
+	}
+	return ""
+}
+
+// EastAsiaFont returns the name of paragraph font family for East Asia.
+func (p ParagraphProperties) EastAsiaFont() string {
+	if fonts := p.x.RPr.RFonts; fonts != nil {
+		if fonts.EastAsiaAttr != nil {
+			return *fonts.EastAsiaAttr
+		}
+	}
+	return ""
+}
+
+// ColorStr returns the hex value of paragraph color.
+func (p ParagraphProperties) ColorStr() string {
+	if color := p.x.RPr.Color; color != nil {
+		valAttr := color.ValAttr
+		if valAttr.ST_HexColorRGB != nil {
+			return *valAttr.ST_HexColorRGB
+		}
+	}
+	return ""
+}
+
+// SpacingValue returns the value of paragraph spacing.
+func (p ParagraphProperties) SpacingValue() int64 {
+	if spacing := p.x.RPr.Spacing; spacing != nil {
+		valAttr := spacing.ValAttr
+		if valAttr.Int64 != nil {
+			return *valAttr.Int64
+		}
+	}
+	return int64(0)
+}
+
+// SpacingMeasure returns the measure of paragraph spacing.
+func (p ParagraphProperties) SpacingMeasure() string {
+	if spacing := p.x.RPr.Spacing; spacing != nil {
+		valAttr := spacing.ValAttr
+		if valAttr.ST_UniversalMeasure != nil {
+			return *valAttr.ST_UniversalMeasure
+		}
+	}
+	return ""
+}
+
+// SizeValue returns the value of paragraph font size.
+func (p ParagraphProperties) SizeValue() float64 {
+	if sz := p.x.RPr.Sz; sz != nil {
+		valAttr := sz.ValAttr
+		if valAttr.ST_UnsignedDecimalNumber != nil {
+			return float64(*valAttr.ST_UnsignedDecimalNumber) / 2
+		}
+	}
+	return 0.0
+}
+
+// SizeMeasure returns the measure of paragraph font size.
```

Comment:
What values can this take?  Any references/urls?

Reply:
fixed